packages/react-icons/convert.js

```diff
@@ -135,10 +135,10 @@ function processFolder(srcPath, destPath, resizable) {
       var jsCode = 
 `
 
-const ${destFilename}Icon = (props: FluentIconsProps) => {
-  const { fill: primaryFill = 'currentColor', className } = props;
```

Comment:
wasn't className used before? I can see it being removed from the destructuring only.